src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -505,12 +510,12 @@ public VisualEditorPageObject typeInParameterField(int i, String parameter) {
   public VisualEditorPageObject clickApplyChanges() {
```

Comment:
I'm wondering about renaming this method to applyChanges() as it does more than just a click.